charts/datadog/values.yaml

```diff
@@ -36,6 +36,20 @@ datadog:
   ## If set, this parameter takes precedence over "appKey".
   appKeyExistingSecret:  # <DATADOG_APP_KEY_SECRET>
 
+  ## Configure the secret backend feature https://docs.datadoghq.com/agent/guide/secrets-management
+  ## Examples: https://docs.datadoghq.com/agent/guide/secrets-management/#setup-examples-1
+  secretBackend:
+    # datadog.secretBackend.command -- Configure the secret backend command, path to the secret backend binary.
+    ## Note: If the command value is "/readsecret_multiple_providers.sh" the agents will have permissions to get secret objects.
+    ## Read more about "/readsecret_multiple_providers.sh": https://docs.datadoghq.com/agent/guide/secrets-management/#script-for-reading-from-multiple-secret-providers-readsecret_multiple_providerssh
+    command:  # "/readsecret.sh" or "/readsecret_multiple_providers.sh" or any custom binary path
```

Comment:
```suggestion
    command: # "/readsecret.sh" or "/readsecret_multiple_providers.sh" or any custom binary path
```

Reply:
the extra space is required by the linter